custom_components/tuya_local/devices/eeese_anna_dehumidifier.yaml

```diff
@@ -0,0 +1,170 @@
+name: Eeese Anna dehumidifier
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+      mapping:
+        - dps_val: false
+          icon: "mdi:air-humidifier-off"
+          icon_priority: 2
+        - dps_val: true
+          icon: "mdi:air-humidifier"
+          icon_priority: 4
+    - id: 2
+      name: humidity
+      type: integer
+      range:
+        min: 30
+        max: 80
+      mapping:
+        - step: 5
+    - id: 5
+      name: mode
+      type: string
+      mapping:
+        - dps_val: Manual
+          value: normal
+        - dps_val: Auto
+          value: auto
+        - dps_val: Laundrydrying
+          value: laundry
+          icon: "mdi:tshirt-crew"
+          icon_priority: 3
+        - dps_val: Sleeping
+          value: sleep
+          icon: "mdi:sleep"
+          icon_priority: 3
+        - dps_val: Purifying
+          value: purify
+          icon: "mdi:air-purifier"
+          icon_priority: 3
+    - id: 6
+      type: integer
+      name: current_humidity
+    - id: 19
+      type: bitfield
+      name: error
```

Comment:
Recently I try to standardise this naming to "fault_code" (and corresponding binary sensor called "Fault")